README.md

```diff
@@ -43,33 +43,38 @@ You can find examples of SBT setup in section [Demo Projects and Examples](#demo
 If you're using [Mill](https://com-lihaoyi.github.io/mill/):
 
 ```scala
+//| mill-version: 1.0.3
+
+package build
+import mill._, scalalib._
+
 object yourProject extends ScalaModule {
-  def scalaVersion = "3.0.0"
+  def jvmId = "temurin:22.0.2" // JavaFX 24 requires JDK version >= 22
+  def scalaVersion = "3.7.2"
 
   // Customize coursier resolution to discover the OS-specific artifacts required by JavaFX
-  // Note: this requires mill >= 0.9.7 (with pr/775 merged)
-  def resolutionCustomizer = T.task {
+  def resolutionCustomizer = Task.Anon {
```

Comment:
Can you please test, if this customizer is still needed? I think it should now work without.

Reply:
Hello, yes I will do so as soon as I get a chance 👍

Reply:
can confirm, it builds and run without those at https://github.com/rom1dep/scalafx-millproject/tree/master

Reply:
Looks unrelated to me, but I don't use ScalaFX, so just guessing. The customized ensured we set the right environment properties when choosing and downloading the transitive dependencies. Since Mill 0.12, we now infer those properties from the environment, like Maven or Gradle does. While being more convenient, this can make the build less reproducible on other platform.

Reply:
Quick addendum: I was able to get the assembly build to run successfully in my Distrobox container after installing a jdk sytstem-wide matching the one specified in the build.mill 👍